package.json

```diff
@@ -78,6 +78,8 @@
     "coveralls": "2.11.2",
     "dateformat": "^1.0.8",
     "ejs": "^2.3.3",
+    "eslint": "^1.5.0",
```

Comment:
`eslint-plugin-markdown` has a peer dependency on ESLint, as most (all?) plugins do.  I suggested finding a way to avoid this in https://github.com/eslint/eslint-plugin-markdown/issues/10, but it was decided to leave as-is.